how-it-works/00_General/01_overview-of-the-internet-computer.card.md

```diff
@@ -6,36 +6,31 @@ title: Architecture of the Internet Computer
 
 # Architecture of the Internet Computer
 
-The Internet Computer (IC) realizes the vision of a *World Computer* – an open and secure *blockchain-based network* that can host programs and data in the form of smart contracts, perform computations on smart contracts in a secure and trustworthy way, and scale infinitely.
-To achieve this, blockchain technology – the foundation of the IC – had to be reconceived from ground up, building on many lessons learnt from earlier blockchain projects.
+The Internet Computer (IC) realizes the vision of a *World Computer* – an open and secure blockchain-based network that hosts programs and data in the form of smart contracts and scales to unbounded capacity.
+To achieve this, blockchain technology – the foundation of the IC – had to be reconceived from ground up, building on many lessons learned from earlier blockchain architectures.
 
-Smart contracts on the Internet Computer are called *canister smart contracts*, or *canisters*, each consisting of a bundle of [*WebAssembly (Wasm)*](https://en.wikipedia.org/wiki/WebAssembly) bytecode and smart contract data storage.
-Each canister has its own, isolated, data storage that is only changed when the canister executes code.
+The Internet Computer hosts smart contracts in the format of *canisters*, each consisting of a [*WebAssembly (Wasm)*](https://en.wikipedia.org/wiki/WebAssembly) bytecode module and data storage.
```

Comment:
I would prefer: The Internet Computer represents smart contracts in the form of *canisters*, ...

Why? We already say the IC *hosts* programs, and we say these programs are smart contracts, so we are re-suing the term "host" too mnay times (and are saying a smart contracts hosts canisters, if you expand the macros).
Also, I prefer "in the form" to the ueber-technical "in the format"